src/components/include/security_manager/ssl_context.h

```diff
@@ -103,6 +104,8 @@ class SSLContext {
                        size_t* out_data_size) = 0;
   virtual bool IsInitCompleted() const = 0;
   virtual bool IsHandshakePending() const = 0;
+  virtual bool GetCertificateDueDate(time_t& due_date) const = 0;
```

Comment:
@AByzhynar description required.

Reply:
@LuxoftAKutsan Changed in eb15dd852a0ea418bfa444b74c8f15d792e254ad